### PR TITLE
Create dscbmr.json

### PR DESCRIPTION
**Pull request by RAYXSTYDIOZ**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] You're not using Vercel or Netlify.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/dscbmr.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "owner": {
+    "username": "RAYXSTYDIOZ",
+    "email": "syedyaseeralirayan@gmail.com",
+    "discord": {
+      "id": "1130109258017550357",
+      "name": "_discord.dscbmr.is-a.dev"
```
[the remark this answers is not in the document]
Comment (TobezDev) [reply domains/dscbmr.json:7]: Correct formatting:
```
{
  "owner": {
    "username": "RAYXSTYDIOZ",
    "email": "syedyaseeralirayan@gmail.com", 
    "discord":  "1130109258017550357"
  },
  "record": {
    "TXT": "dh=af0561dc7d2f7eb0ef153cb3a270c0b98fe5c36e"
  }
}
```

Summary:
* Changed `discord` object:
  - removed `username`
  - set `discord` to the user's id to follow documentation
 * reformatted file to use new `.prettierrc`

[the remark this answers is not in the document]
Comment (RAYXSTYDIOZ) [reply domains/dscbmr.json:7]: thank you

[the remark this answers is not in the document]
Comment (RAYXSTYDIOZ) [reply domains/dscbmr.json]: I UPDATED FILES AND THANK YOU FOR HELPING ME IM A BEGINNER

File: domains/dscbmr.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "RAYXSTYDIOZ",
+    "email": "syedyaseeralirayan@gmail.com", 
+    "discord":  "1130109258017550357"
+  },
+  "record": {
+    "TXT": "dh=af0561dc7d2f7eb0ef153cb3a270c0b98fe5c36e"
```
Comment (CuteDog5695) [inline domains/dscbmr.json:8]: Not the way to connect your domain to Discord, check here https://www.is-a.dev/docs/discord_verification/

**Review by iostpa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**